docs/content/_navigation.json

```diff
@@ -316,6 +316,10 @@
             "title": "Deploying Dagster to AWS",
             "path": "/deployment/guides/aws"
           },
+          {
+            "title": "Deploying Dagster to ECS",
+            "path": "/deployment/guides/ecs"
+          },
```

Comment:
this is a bit weird since ECS is of course a subset of AWS, but it seems bad to not have some top-level link to the ECS part of the docs